app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryFragment.java

```diff
@@ -153,16 +156,11 @@ public void showBooks(LinkedList<Book> books) {
       displayNoItemsAvailable();
       return;
     }
-
+    this.books=books;
     Log.i("kiwix-showBooks", "Contains:" + books.size());
     libraryAdapter.setAllBooks(books);
-    if (faActivity.searchView != null) {
-      libraryAdapter.getFilter().filter(
-          faActivity.searchView.getQuery(),
-          i -> stopScanningContent());
-    } else {
-      libraryAdapter.getFilter().filter("", i -> stopScanningContent());
-    }
+    String query = (faActivity.searchView != null)?faActivity.searchView.getQuery().toString():"";
```

Comment:
Please refactor this. Ternaries are not amazing and this is bad style as you could use == null and invert. Spacing as well.